src/components/MessageSuggestions.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback } from "react";
+import { useChatActions } from "@yext/chat-headless-react";
+import { useDefaultHandleApiError } from "../hooks/useDefaultHandleApiError";
+import { withStylelessCssClasses } from "../utils/withStylelessCssClasses";
+import { useComposedCssClasses } from "../hooks";
+
+/**
+ * The CSS class interface for the {@link MessageSuggestions} component.
+ *
+ * @public
+ */
+export interface MessageSuggestionClasses {
+  container?: string;
+  suggestion?: string;
+}
+
+/**
+ * The props for the {@link MessageSuggestions} component.
+ *
+ * @public
+ */
+export interface MessageSuggestionsProps {
+  suggestions: string[];
+  customCssClasses?: MessageSuggestionClasses;
+}
+
+const defaultClassnames = withStylelessCssClasses("Suggestions", {
+  container: "flex gap-2 mt-4 w-full overflow-x-auto flex-wrap",
+  suggestion:
+    "hover:cursor-pointer px-2 py-1 bg-white hover:bg-slate-300 rounded-full text-sm text-blue-700 border border-blue-700 hover:underline",
+}) satisfies MessageSuggestionClasses;
+
+/**
+ * A component that displays a list of suggested messages
+ * to the user, which they can click to send the message to the bot.
+ *
+ * @public
```

Comment:
is this component actually public? I don't think we need to export it for users to use so this really can be `@internal`